midealocal/devices/db/__init__.py

```diff
@@ -26,6 +38,134 @@ class DeviceAttributes(StrEnum):
+        0x07: "1600",
+        0x08: "1300",
+        0xFF: "default",
+    }
+
+    _water_level: ClassVar[dict[int, str]] = {
+        0x01: "Low",
+        0x02: "Mid",
+        0x03: "High",
+        0x04: "4",
+        0x05: "Auto",
+    }
+
+    _program: ClassVar[dict[int, str]] = {
+        0x00: "cotton",
+        0x01: "eco",
+        0x02: "fast_wash",
+        0x03: "mixed_wash",
+        0x04: "fiber",
+        0x05: "wool",
+        0x06: "enzyme",
+        0x07: "ssp",
+        0x08: "sport_clothes",
+        0x09: "single_dehytration",
+        0x0A: "rinsing_dehydration",
+        0x0B: "big",
+        0x0C: "baby_clothes",
+        0x0D: "outdoor",
+        0x0E: "air_wash",
+        0x0F: "down_jacket",
+        0x10: "color",
+        0x11: "intelligent",
+        0x12: "quick_wash",
+        0x13: "kids",
+        0x14: "water_cotton",
+        0x15: "single_drying",
+        0x16: "single_drying",
+        0x17: "fast_wash_30",
+        0x18: "fast_wash_60",
+        0x19: "water_intelligent",
+        0x1A: "water_steep",
+        0x1B: "water_fast_wash_30",
+        0x1C: "shirt",
+        0x1D: "steep",
+        0x1E: "new_water_cotton",
+        0x1F: "water_mixed_wash",
+        0x20: "water_fiber",
+        0x21: "water_kids",
+        0x22: "water_underwear",
+        0x23: "specialist",
+        0x24: "water_eco",
+        0x25: "wash_drying_60",
+        0x26: "self_wash_5",
+        0x27: "fast_wash_min",
+        0x28: "mixed_wash_min",
+        0x29: "dehydration_min",
+        0x2A: "self_wash_min",
+        0x2B: "baby_clothes_min",
+        0x2C: "prevent_allergy",
+        0x2D: "cold_wash",
+        0x2E: "soft_wash",
+        0x2F: "remove_mite_wash",
+        0x30: "water_intense_wash",
+        0x31: "fast_dry",
+        0x32: "water_outdoor",
+        0x33: "spring_autumn_wash",
+        0x34: "summer_wash",
+        0x35: "winter_wash",
+        0x36: "jean",
+        0x37: "new_clothes_wash",
+        0x38: "silk",
+        0x39: "insight_wash",
+        0x3A: "fitness_clothes",
+        0x3B: "mink",
+        0x3C: "fresh_air",
+        0x3D: "bucket_dry",
+        0x3E: "jacket",
+        0x3F: "bath_towel",
+        0x40: "night_fresh_wash",
+        0x50: "water_fiber",
+        0x51: "diy0",
+        0x52: "diy2",
+        0x60: "heart_wash",
+        0x61: "water_cold_wash",
+        0x62: "water_prevent_allergy",
+        0x63: "water_remove_mite_wash",
+        0x64: "water_ssp",
+        0x65: "silk_wash",
+        0x66: "standard",
+        0x67: "green_wool",
+        0x68: "cook_wash",
+        0x69: "fresh_remove_wrinkle",
+        0x6A: "steam_sterilize_wash",
+        0x6B: "aromatherapy",
+        0x70: "sterilize_wash",
+        0xFE: "love",
+        0xFF: "default",
+    }
+
+    _temperature: ClassVar[dict[int, str]] = {
+        0x01: "0",
+        0x02: "20",
+        0x03: "30",
+        0x04: "40",
+        0x05: "60",
+        0x06: "95",
+        0x07: "70",
```

Comment:
I think the 2 values are swapped, the progress is rising

Reply:
this is the origina value from lua script.